README.md

```diff
@@ -1,23 +1,63 @@
+Thus, quality estimation for a processed video sequence requires access to the original, high quality video sequence. 
+
+This software has not been maintained since 2013 and is provided for archival and research purposes only. 
+ITS does not have the funding to perform troubleshooting. 
+All souce code was developed for MATLAB® R2013b (8.2). Future versions of MATLAB may cause the code not to run.
+
+These tools are available free of charge for any commercial or non-commercial use, in accordance with the terms described in the [LICENSE](https://github.com/NTIA/vqm/blob/master/LICENSE.md).
 
-The available software packages are:
+More information on these tools, including publications that fully document the VQM metrics and INLSA algorithm, can be found on the [ITS website](https://its.ntia.gov/research-topics/video-quality-research/software/).
 
-| Name | Title |
+| Tool | Description |
 | ---- | ----- |
 | `bvqm` | Batch Video Quality Metric |
 | `cvqm` | Command Line Video Quality Metric |
 | `fdf` | Estimate HRC Fraction Dropped Frames using RR and NR VQMs |
 | `psnr` | Peak Signal to Noise Ratio Search and PSNR Variable Frame Delay |
 | `vfd` | Variable Frame Delays |
+| `inlsa` | Iterated Nested Least Squares Algorithm, for combining multiple datasets |
+| `screening` | Methods to screen subjects in a subjective test |
+| `si` | Spatial Information Filter |
+
+
+## Video Quality Metrics: BVQM, CVQM, FDF, PSNR, and VFD
+
+The VQM software tools provide both standardized and non-standardized methods for measuring the video quality of digital video systems. 
+Each VQM tool estimates how people perceive video quality. 
+These VQM algorithms compare the processed video (output) with the original video (input). 
+These models are suitable when the original video is good quality or better. 
+Quality problems from the camera and original production are not considered. 
+
+Although ITS released several metrics, the most impactful metrics for video quality assessment are as follows:
+
+- NTIA General Model, referred to as 'VQM' in literature, as given in the ANSI T1.801.03-2003, ITU-T J.144 (03/04), ITU-R BT.1683 (06/04), [this report](https://its.ntia.gov/publications/details.aspx?pub=2423), and [this journal article](https://its.ntia.gov/publications/details.aspx?pub=2576). Finalized in 2001.
+- NTIA Developers Model, a fast running variant of the General Model, described in [this report](https://its.ntia.gov/publications/details.aspx?pub=2423). Finalized in 2001.
+- NTIA Video Quality Model for Variable Frame Delay (VQM_VFD), described in [this report](https://its.ntia.gov/publications/details.aspx?pub=2556). Finalized in 2011.
+
+The Batch Video Quality Metric (BVQM) Software performs out-of-service, lab bench testing. 
+BVQM can processing and analyses of multiple video scenes and multiple video systems at once. 
```

Comment:
BVQM can processing and analyses of multiple > BVQM can process and analyze multiple